src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -108,18 +137,11 @@
-                        if cleanup_task is not None:
-                            logger.info("Cancelling %s cleanup task", task_name)
-                            cleanup_task.cancel()
-                        else:
-                            logger.warning("No cleanup task found for %s", task_name)
-
-                    self._cleanup_tasks.clear()
+                if self._job_store is not None:
+                    try:
+                        await self._job_store.close()
+                    except Exception as e:
+                        logger.exception("Error closing Dask client: %s", e)
```

Ruff (0.12.2)

Redundant exception object included in `logging.exception` call

(TRY401)